src/components/pages/service/support.tsx

```diff
@@ -36,16 +36,30 @@ const ContentStyled = styled.div`
   ${TextStyles.textR}
 `;
 
-export const Intro = ({ illustration, headline, children }: any) => {
+const Kicker = styled.span`
+  ${TextStyles.toplineS}
+  display: block;
+  margin-bottom: 12px;
+  color: #3e61ee;
```

Comment:
I know there are many other places in the code where we hardcode the colors. But in general it is better to use the colors from the theme file. If we want to change them later, it will be much faster:
`color: ${theme.palette.text.topline};`

Reply:
i completely agree with you, the use of theme's constants is much better